Engine/source/gui/worldEditor/worldEditor.h

```diff
@@ -92,7 +92,6 @@ class WorldEditor : public EditTSCtrl
       S32 getSelectionSize();
       S32 getSelectObject(S32 index);	
       const Point3F& getSelectionCentroid();
-      const char* getSelectionCentroidText();
```

Comment:
So if I recall correctly the only reason this was here is to output to a console method, after converting it over to use a return console type of Point3F it isn't really needed. Guess this isn't so much documentation, but in the act of converting over console functions to use the new way of defining them, it was a necessary change. Hopefully that all makes sense. Just look at line 3252 or 3261 in the .cpp file.